CARTOGRAPHY.md

```diff
@@ -37,3 +37,51 @@ account.
+
+* The connection between data and appearance of features in the map needs to be comprehensible for the viewer.
+
+* When changes have a strong impact on the overall appearance of the map input should be sought from the broader OpenStreetMap community and particular consideration given to their comments and concerns.
+
+### Colors
+
+* Differentiating features through different colors only makes sense if this difference can actually be read by the map users.
+
+* For area colors we prefer light and low saturation tones.  Lines, individual symbols and labels also can be stronger in color.
+
+* Stronger colors indicate fairly distinct and meaningful features as opposed to more general and less distinct features which use weaker colors.
+
+* Small areas work well with somewhat stronger colors while large areas tend to call for weaker tones.
+
+* Symbols of area patterns need to be relatively weak in color with little contrast towards the base color so they cannot be mistaken for an individual symbol.
+
+* Area outlines should - when used - be in colors that are not too strong and harmonic with the fill color.  Care should be taken that area outlines are not mistaken for line features.
+
+* Use of blue for area and line colors should be reserved for water related features.  Symbols and labels in blue color can also be used for other purposes but sparsely and in a tone clearly distinct from all water related features
+
+* Use of green should be reserved for vegetation related features, in particular in case of area and line colors.
+
+* Use of red, orange, yellow and brown line colors should be reserved to roads and paths.
+
+* Transparency tends to lead to undesirable and ambiguous color mixing and should therefore be used very carefully.
+
+* It is generally advisable to design color relationships in perceptual color spaces.
+
+### Zoom levels
+
+* Styling of map elements should work for the entire range of displayed latitudes and the full bandwidth of geographic situations that exists on earth.
+
+* Design differences between subsequent zoom levels should be small.  Continuous or small step changes over multiple levels is preferable in comparison to constant styling across multiple levels followed by a huge step.
+
+* Size increase (line widths, text sizes, icon sizes etc.) with increasing zoom level should not be larger than the scale ratio between the zoom levels so sizes do not increase relative to ground units at higher zoom levels.
+
+* Features can start to appear at a certain zoom level when zooming in but they should not disappear again at a later zoom level - except possibly for labels of features becoming very large.
+
+* Starting zoom levels for showing features should be selected so that competition between elements does not result in the majority of features of a certain type not being shown if there is no meaningful measure of priority that is used for selection.
```

Comment:
If you'd for example show all shops at z16 that would mean only a small fraction of the shops are actually shown (_the majority of features of a certain type not being shown_) because Mapnik does not render a symbol if there is already one rendered that would be overlapping.  If the rendering order or prioritization of the shops is not meaningful (_there is no meaningful measure of priority_) the results would be highly non-intuitive which is not desirable.